app/views/about/privacy.en.html.erb

```diff
@@ -1,155 +1,154 @@
+        </li>
+        <li>
+          your Internet Protocol (IP) address
+        </li>
+        <li>
+          details of which version of
+          <a class="govuk-link" rel="external" target="_blank" href="https://www.gov.uk/support/browsers/">web browser</a>
+          you used
+        </li>
+        <li>
+          information on how you use the site, using <a class="govuk-link" target="_blank" href="/cookies">cookies</a> and page tagging techniques
+        </li>
       </ul>
     </div>
-    <p class="govuk-body">The LAA is an Executive Agency of the Ministry of Justice (MoJ). The MoJ is the data controller for the personal information we hold.</p>
-    <p class="govuk-body">The LAA collects and processes personal data for the exercise of its own and associated public functions. Our public function is to provide legal aid.</p>
 
-    <h2 class="govuk-heading-l">About personal information</h2>
-    <p class="govuk-body">Personal data is information about you as an individual. It can be your name, address or telephone number. It can also include the information that you've provided in this service such as:</p>
+    <p class="govuk-body">Where you provide your consent, we use
+      <a class="govuk-link" rel="external" target="_blank" href="https://support.google.com/analytics/topic/2919631/">Google Analytics</a>
+        cookies. Google Analytics processes information about:
+    </p>
     <div class="govuk-list">
       <ul class="govuk-list govuk-list--bullet">
-        <li>your financial circumstances</li>
-        <li>information relating to any current or previous legal proceedings concerning you</li>
+        <li>the pages you visit in this service</li>
+        <li>how long you spend on each page</li>
+        <li>how you got to this service</li>
+        <li>what you click on while you’re using the service</li>
       </ul>
     </div>
-    <p class="govuk-body">We know how important it is to protect customers’ privacy and to comply with data protection laws. We will safeguard your personal data and will only disclose it where it is lawful to do so, or with your consent.</p>
 
-    <h2 class="govuk-heading-l">Types of personal data we process</h2>
-    <p class="govuk-body">We only process personal data that is relevant for the services we provide.</p>
-    <p class="govuk-body">The personal data which you've provided for this service will only be used for the purposes set out below.</p>
+    <p class="govuk-body">We make sure you cannot be identified by Google Analytics data. We do not pass any of your personal data to Google Analytics, and Google Analytics does not store your IP address.</p>
+    <p class="govuk-body">We will not combine analytics information with other data sets in a way that would identify who you are.</p>
+    <p class="govuk-body">Find out more about <a class="govuk-link" target="_blank" href="/cookies">how we use Google Analytics and other cookies</a>
```

Comment:
Can we use the rails path helper? Also better not to be open in new page/tab unless there is a very good reason for, and if so we need to say something like `(opens in new tab)`

Reply:
I was following what was already there. V happy to change.